### ISSUE TITLE
[A11y Library Fix] - eCR Library wrapper is inaccessible when no eCRs are found

### ISSUE THREAD

**Issue by jakewheeler** [CDCgov/dibbs-ecr-viewer#397]
## What needs to be done

When no eCRs are present in the library, the Axe extension will produce an a11y issue that the scrollable region (`.ecr-library-wrapper`) must be accessible by keyboard. This happens because there are no focusable elements within the container when no eCRs are found. We should investigate this issue further and implement an adequate solution.

Please see the [full issue report](https://cdc-devtools.dequecloud.com/issues/236d3d9b-5ef1-4a66-8320-8ac222126982) for additional details. Please note that the "Help` section at the bottom offers a couple of potential solutions.

## Why it needs to be done

Keyboard users are currently unable to access the content within the `.ecr-library-wrapper` element when no eCRs are are available.

## Acceptance Criteria

- [ ] A11y issue has been reviewed and addressed appropriately
